test/utils.rb

```diff
@@ -111,13 +111,18 @@ def issue_crl(revoke_info, serial, lastup, nextup, extensions,
     crl
   end
 
-  def get_subject_key_id(cert)
+  def get_subject_key_id(cert, hex: true)
```

Comment:
`unpack_subject_key_id` might be better?

Reply:
I'm just updating the existing method here. You'd like me to change the name?

Reply:
I feel like `get_subject_key_id` is not a great name, but I'll leave it up to you. It's a relatively minor point.